src/web/hello_js/hello_js.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Hello JS
+level: 1
+author: Arve Seljebu
+---
+
+# Introduksjon {.intro}
+I denne oppgaven blir du introdusert for programmeringsspråket JavaScript. Du
+skal gjøre den klassiske oppgaven *Hello world*, med en katt.
+
+![](jsbin.png)
+
+# Steg 1: Bruke JS Bin {.activity}
+Du skal bruke en nettside som heter *JS Bin* for denne oppgaven. Dette fordi
+JS Bin ser likt ut i alle nettlesere, så da er det enklere å følge stegene.
+
+## Sjekkliste {.check}
+
+- Åpne [JS Bin](http://jsbin.com/?js,console) i et eget vindu.
+- Legg merke til at du får to faner; **JavaScript** og **Console**.
+- Koden skrives i **JavaScript**.
+- Og i **Console** vises tekstbeskjeder som blir skrevet ut.
+- Skriv inn denne koden:
+
+  ```js
+  console.log("hello world");
+  ```
+
+- Trykk **Ctrl** + **Enter** (eller bruk knappen <button>Run</button>) og se hva som skjer
+  i **Console**.
+- Kan du endre koden til å skrive ditt navn?
+
+## Forklaring {.tip}
+Her er en forklaring til koden over:
+- `console.log()` betyr at vi ønsker å kjøre kommandoen med navnet `console.log`.
+- `.log` er en av flere utskriftsmetoder. Prøv `console.error`, ser du hva
+  forskjellen til `console.log` er?
+- Tekst skrives mellom `"`, slik som `"hello world"`.
+- `;` betyr at kodesetningen er ferdig.
+
+# Steg 2: Lage en funksjon {.activity}
+En funksjon er nesten som en oppskrift. Funksjoner brukes når man ønsker å
+gjenbruke kode flere ganger eller bryte opp et problem til mindre biter. En
+funksjon kan både ta i mot og returnere data. Din funksjon skal hete `hei`,
+ta i mot et `navn` og skrive navnet til konsollen.
+
+## Sjekkliste {.check}
+
+- Slett koden din fra steg 1.
+- Skriv inn denne koden:
+
+  ```js
+  function hei (navn) {
```

Comment:
Dette er stilen ["standard"](http://standardjs.com/), pluss semikolon (siden JS Bin klager på det). Tok meg friheten siden dette er første JS-oppgave. Vet det ser litt rart ut, men det har fordelen av å skille på definisjon `fn ()` og kall `fn()` av funksjoner, som kanskje er fint for nybegynnere?